commit 6c051d7e39aec90927fb00849a523452ef768b2c (refs/changes/60/3460/2)
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   2018-02-23T20:55:03+00:00 (1 year, 7 months ago)
    
    OS-6689 illumos-extra.manifest not regenerated when projects/illumos-extra/manifest changes

diff --git a/Makefile b/Makefile
index 8b79c2f5..00098ebb 100644
--- a/Makefile
+++ b/Makefile
@@ -206,7 +206,8 @@ $(MCPROTO)/illumos.mancheck.conf: projects/illumos/mancheck.conf | $(MCPROTO)
 $(BOOT_MPROTO)/illumos.manifest: projects/illumos/manifest | $(BOOT_MPROTO)
 	cp projects/illumos/boot.manifest $(BOOT_MPROTO)/illumos.manifest
 
-$(MPROTO)/illumos-extra.manifest: 1-extra-stamp | $(MPROTO)
+$(MPROTO)/illumos-extra.manifest: 1-extra-stamp \
+    projects/illumos-extra/manifest | $(MPROTO)
 	gmake DESTDIR=$(MPROTO) DESTNAME=illumos-extra.manifest \
 	    -C projects/illumos-extra manifest; \
 
